src/Domain/Package/PackageManager.php

```diff
@@ -205,7 +216,12 @@ public function getAlterData(): array {
   private function initializePackages(FixturePathHandler $fixture_path_handler, string $packages_config, ?string $packages_config_alter): void {
     $data = $this->parseYamlFile($this->orca->getPath($packages_config));
     if ($packages_config_alter) {
-      $alter_path = $this->orca->getPath($packages_config_alter);
+      if (!$this->filesystem->isAbsolutePath($packages_config_alter)) {
+        $alter_path = $this->orca->getPath($packages_config_alter);
+      }
+      else {
+        $alter_path = $packages_config_alter;
+      }
```

Comment:
Just for readability, let's reverse this condition: (from "if not then don't" to "if then")

```suggestion
      if ($this->filesystem->isAbsolutePath($packages_config_alter)) {
        $alter_path = $packages_config_alter;
      }
      else {
        $alter_path = $this->orca->getPath($packages_config_alter);
      }
```